### PR TITLE
Add Spectron

### PR DESCRIPTION
**Pull request by FrederikBolding**

### CODE REVIEW

**Comment by FrederikBolding** [comment]
[ch7034]

File: jest_config/jest.config.e2e.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
```
Comment (Mrtenz) [inline jest_config/jest.config.e2e.json:1]: Why have a separate config for E2E tests? Can we not combine this into one config?

File: e2e/basic-electron.spec.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    return app.start();
+  });
+
+  afterAll(() => {
+    if (app && app.isRunning()) {
+      return app.stop();
+    }
+  });
+
+  it('Opens a window', async function () {
```
Comment (Mrtenz) [inline e2e/basic-electron.spec.ts:16]: Test names shouldn't start with a capital letter. Also we can use an arrow function here.

```suggestion
  it('opens a window', async () => {
```

File: e2e/basic-electron.spec.ts [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    const count = await app.client.getWindowCount();
+    expect(count).toEqual(1);
+  });
+
+  it('Has the correct title', async () => {
```
Comment (Mrtenz) [inline e2e/basic-electron.spec.ts:21]: Test names shouldn't start with a capital letter.

File: e2e/basic-electron.spec.ts [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    const title = await app.client.getTitle();
+    expect(title).toEqual('Signer');
+  });
+
+  it('Renders the initial view', async () => {
```
Comment (Mrtenz) [inline e2e/basic-electron.spec.ts:26]: Test names shouldn't start with a capital letter.

**Review by Mrtenz — CHANGES_REQUESTED** [review]

File: jest_config/jest.config.e2e.json (hunk 1 above)
Comment (FrederikBolding) [reply jest_config/jest.config.e2e.json:1]: Maybe? I just couldn't figure out how to run them on separate commands without

**Review by Mrtenz — APPROVED** [review]

**Pull request merged**